<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1341</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>0</y>
      <width>1311</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>image originale</string>
    </property>
    <widget class="QLabel" name="img1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>301</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="histimg1">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>671</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="parcourir">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>parcourir</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>240</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Image Originale</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>240</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>histogramme de l'Image Originale</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>1311</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Egalisation d'histogramme</string>
    </property>
    <widget class="QLabel" name="img2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>301</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="histimg2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>671</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="appliquer">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>appliquer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>240</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Image egalisée</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>240</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>histogramme de l'Image filtrée</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>540</y>
      <width>581</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>seuillage</string>
    </property>
    <widget class="QLabel" name="img4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>321</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="valider2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>valider</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="binary">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>seuillage binaire</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="otsu">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>seuillage d'Ostu</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>230</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Image seuillée</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>540</y>
      <width>581</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Filtrage</string>
    </property>
    <widget class="QLabel" name="img3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>331</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="valider1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>valider</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="gaussian">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Filtre gaussien</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="average">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Filter moyenneur</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="median">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Filtre median</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>230</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Image filtrée</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1341</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
